packages/frontend/src/hooks/useAllTimeStats.tsx

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from '@tanstack/react-query';
+import { request, gql } from 'graphql-request';
+import GRAPH_URLS from 'src/subgraph.json';
+
+const ALL_TIME_STATS_QUERY = gql`
+  query {
+    allTimeStats(id: "current") {
+      volumeUsd
+    }
+  }
+`;
+
+type AllTimeStatsResponse = {
+  allTimeStats: {
+    volumeUsd: string;
+  };
+};
+
+const useAllTimeStats = () => {
```

Comment:
If you're doing it in serial as a rate limiter, cool. If not:

```typescript
const responses = await Promise.all(Object.values(GRAPH_URLS).map(url => request(...).catch((error: any) => {
  console.error('....');
});

const totalVolumeUsd = responses.reduce((sum, response) => Number(response?.allTimeStats?.volumeUsd ?? 0), 0);

const totalVolumeUsdFormatted = Math.round(totalVolumeUsd);
```

Reply:
Oh, concurrent is clearly the better option. Much faster now.